content/src/cloud/remote-ui/remote-ui-deep-dive.md

```diff
@@ -33,17 +33,20 @@ To handle simultaneous requests, all data is routed through a TCP multiplexer. T
 2. Decrypting the packets using the SSL certificate.
 3. Forwarding the decrypted data to the HTTP component for further handling.
 
-## Caveats
+## Limitations
 
-We are currently not forwarding the IP address of the incoming request. Because of this, we are unable to support Home Assistant instances that have configured `127.0.0.1` or `::1` as trusted networks or proxies. It also means that if you use IP bans, the remote connection will be banned as a whole instead of just the address from which the incorrect passwords were entered. We are currently exploring a solution for this issue.
+We are currently not forwarding the IP address of the incoming request. This means that the Remote UI cannot be used if you are using one of the following network scenarios.
+
+- If your Home Assistant instances that has configured `127.0.0.1` or `::1` as trusted networks or proxies.
```

Comment:
This should link to the http integration were you define this.

Reply:
Thank you for the catch! https://github.com/NabuCasa/support/pull/363